docs/website/docs/general-usage/credentials/config_providers.md

```diff
@@ -92,12 +92,26 @@ sections and key names are separated by "." into the provider-specific formats.
 1. For TOML, names are case-sensitive and sections are separated with ".".
 1. For Environment Variables, all names are capitalized and sections are separated with double
    underscore "__".
-
-Example: When `dlt` evaluates the request `dlt.secrets["my_section.gcp_credentials"]` it must find
-the `private_key` for Google credentials. It will look
-
-1. first in env variable `MY_SECTION__GCP_CREDENTIALS__PRIVATE_KEY` and if not found,
-1. in `secrets.toml` with key `my_section.gcp_credentials.private_key`.
+   
+   Example: To override a token in "secrets.toml":
+   
+   ```toml
+   [sources.github]
+   access_token = "GITHUB_API_TOKEN"
+   ```
+   Use the following environment variable:
+   ```bash
+   export SOURCES__GITHUB__ACCESS_TOKEN="your_token_here"
+   ```
+
+
+1. When `dlt` evaluates a request like  'dlt.secrets["my_section.gcp_credentials"]', it looks for it as follows: 
```

Comment:
replace this sentence with this please:
When `dlt` evaluates the request `dlt.secrets["my_section.gcp_credentials"]` it must find the `private_key` for Google credentials.  It looks for it as follows:

Reply:
done